helpers.go

```diff
@@ -90,8 +97,8 @@ func getBucketNameFromRequestUri(urlPath string)string{
 	
 	// Adding this because there might be a path for bucket, so grabbing only bucket name
 	bucketName := strings.Split(res[0],"/")[0] 
-	fmt.Println("In BucketName Simple Download")
-	fmt.Println(bucketName)
+	log.Debug("In BucketName Simple Download")
```

Comment:
I am not that familiar with logging in go. But, if we want to have the context of the log (i.e. file/func name), it's probably better done in where the log is  [configured](https://github.com/byronwhitlock-google/go-gcsproxy/blob/main/main.go#L62-L64)